.eslintrc.js

```diff
@@ -53,7 +53,9 @@ module.exports = {
     "import/extensions": [2, {
       js: "never",
       jsx: "never",
-      json: "always"
+      json: "always",
+      ts: "never",
```

Comment:
Should changing this now give an error every time we import a .ts file with extension? I do not see any new errors for files where we are already importing with extension.

Reply:
no, it's the opposite. this didn't impact anything locally, but github actions was throwing an error about that file extension. so something is different about our eslint (or tsconfig?) setup in github actions vs. local.